examples/src/retrievers/multi_vector_hypothetical.ts

```diff
@@ -12,6 +12,7 @@ import { InMemoryStore } from "langchain/storage/in_memory";
 import { TextLoader } from "langchain/document_loaders/fs/text";
 import { Document } from "langchain/document";
 import { JsonKeyOutputFunctionsParser } from "langchain/output_parsers";
+import { createDocumentStoreFromByteStore } from "langchain/storage/encoder_backed";
```

Comment:
The whole idea here was to never see this function again?

Reply:
Whoops, was unaware. Will fix.